README.md

```diff
@@ -46,24 +46,10 @@ Links to presentations, discussions and interviews are available in
 To get started and contribute to the project, visit the [ Technical setup](./TECHNICAL-SETUP.md) page and the
```

Comment:
```suggestion
To get started and contribute to the project, see [Technical Setup](./TECHNICAL-SETUP.md) and the
```

Reply:
it's useful to have the title of the page as the descriptive link test, and the more informal "see" (we don't need to say "page")